tests/foreman/__init__.py

```diff
@@ -0,0 +1,3 @@
+from robottelo.config import settings
+
+settings.configure()
```

Comment:
I find it a little bit curious that `configure()` exists at all. Now take my thoughts with a grain of salt, as I've only read through the class once. But if any configuration file parsing or checking needs to happen before the `settings` object can be used at all, then shouldn't that parsing/checking be done in `Settings.__init__`? That would allow for us to write more terse client code:

```python
from robottelo.config import Settings

Settings().some_setting
```

If performance is a concern, we can implement caching. But let's do some proper measurements before even doing that. Any halfway decent OS will cache files that are read from disk.